scene/resources/animation.h

```diff
@@ -485,7 +498,10 @@ class Animation : public Resource {
 VARIANT_ENUM_CAST(Animation::TrackType);
 VARIANT_ENUM_CAST(Animation::InterpolationType);
 VARIANT_ENUM_CAST(Animation::UpdateMode);
-VARIANT_ENUM_CAST(Animation::HandleMode);
 VARIANT_ENUM_CAST(Animation::LoopMode);
+#ifdef TOOLS_ENABLED
+VARIANT_ENUM_CAST(Animation::HandleMode);
+VARIANT_ENUM_CAST(Animation::HandleSetMode);
+#endif // TOOLS_ENABLED
```

Comment:
These can likely be removed too I think.

Reply:
This is necessary if Enum is used as a function argument. It can be removed if you are using an int as a function argument.

Reply:
Well, I recommend leaving this as is. If you must remove this, please let me know again.

Reply:
Sounds good :)